docs/src/main/tut/docs/parameterized_queries.md

```diff
@@ -10,23 +10,22 @@ title: "Parameterized Queries"
 
 ```scala
 val name = "John"
-val query = c"create (a:Test {name: $name})"
+val query = c"CREATE (a:Test {name: $name})"
 
-assert(query.rawQuery == "create (a:Test {name: $p1})")
+assert(query.rawQuery == "CREATE (a:Test {name: $p1})")
 assert(query.params == Map("p1" -> QueryParam("John")))
 ```
 
 All parameters will be converted to neo4j supported types (please see [Supported types](types.html)).
 
-`c` interpolator creates `DeferredQueryBuilder` which is an immutable representation of a cypher query.
+`c` interpolator CREATEs `DeferredQueryBuilder` which is an immutable representation of a cypher query.
```

Comment:
`CREATEs` seems an auto replacement 😁

Reply:
Oh...
Yeah, you catch me there, sorry.
There were some others, it seems I forget this one.
Let me fix it real quick!